Makefile

```diff
@@ -28,7 +28,7 @@ override CARGO = cross
 endif
 
 ifeq ($(CARGO),cross)
-override TARGET_DIR := $(or $(TARGET_DIR),./target/build/$(TARGET)/)
+override TARGET_DIR := $(or $(TARGET_DIR),./target/)
```

Comment:
**question:** Why has this changed? I think this is the cause of the CI failures.

Reply:
I was getting errors on my local machine for the wrong build targets must be a configuration issue

Reply:
When I run `make test/k8s/cluster-wasmtime`

I get:

```
...
install: cannot stat './target/build/x86_64-unknown-linux-musl/x86_64-unknown-linux-musl/debug/containerd-shim-wasmtime-v1'
```

Reply:
I do not know why this was occurring but after cleaning up the build folders (`target` and `dist`) and doing `cargo clean`.  I no longer have the issue 🤷